pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/model/XPathSuggestions.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.util.fxdesigner.model;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
+
+
+
+
+public class XPathSuggestions {
+    private List<String> xPathSuggestions = new ArrayList<>();
+    private String language;
+    private List<String> listOfSuggestions;
+
+
+    public XPathSuggestions(String language) {
+        this.language = language.replaceAll("[0-9]", "").replaceAll("//s", "").trim().toLowerCase();
```

Comment:
The name you're looking for here is Language.getTerseName. But hand a Language instance to the constructor